.travis.yml

```diff
@@ -1,9 +1,50 @@
-# Use Docker infrastructure
-sudo: false
-
+---
+dist: trusty
 language: ruby
 cache: bundler
-
+before_install:
+  - if [ $BUNDLER_VERSION ]; then
+      gem install -v $BUNDLER_VERSION bundler --no-rdoc --no-ri;
+    fi
+  - bundle -v
+  - rm -f Gemfile.lock
+  - gem update --system $RUBYGEMS_VERSION
+  - gem --version
+  - bundle -v
+script:
+  - 'SIMPLECOV=yes bundle exec rake $CHECK'
+bundler_args: --without system_tests
 rvm:
-  - 2.4.4
-  - 2.1.9
+  - 2.5.1
+env:
+  global:
+    - BEAKER_PUPPET_COLLECTION=puppet6 PUPPET_GEM_VERSION="~> 6.0"
+matrix:
```

Comment:
If I understand correctly, are you using a matrix to get past the following?

Note from Travis
```
Bundler 2.0 #
On January 3rd 2019 the Bundler team released Bundler 2.0 which dropped support for Ruby versions 2.2 and older, and added a new dependency on RubyGems 3.0.0.
```

Reply:
https://github.com/cisco/cisco-network-puppet-module/pull/530/files/2921a5a35cf2627f128b54147d8a90489ae528f3#diff-354f30a63fb0907d4ad57269548329e3R33 is mainly how we are getting past it by specifying `BUNDLER_VERSION=1.17.3` and `RUBYGEMS_VERSION` then during the before install it will install the bundler version if there is one present, other wise it will use the bundler version included in the gemset/rubygems version.